lms/djangoapps/discussion/rest_api/tests/test_views.py

```diff
@@ -1,2027 +1,232 @@
 """
 Tests for Discussion API views
+TODO: Requires a discussion with the team, as these tests seems to be departed.
```

Comment:
What is meant by this?

Reply:
Comment.retrieve_all() wasn't implemented in forum v2 since it wasn't being used anywhere in the lms client and therefore these tests were commented out.  I have now implemented the `retrieve_all()` method and fixed the tests. Alternatively, we can remove the tests completely.